src/middlewares.js

```diff
@@ -0,0 +1,36 @@
+const { verify } = require('jsonwebtoken');
+
+exports.verify = (req, res, next) => {
+  if (!req.headers.cookie || !req.headers.cookie.includes('jwt')) {
+    next();
+  } else {
+    const jwt = req.cookies.jwt;
+    if (jwt) {
+      verify(jwt, process.env.SECRET, (err, jwt) => {
+        if (err) {
+          res.send('Fail');
```

Comment:
Handle invalid token properly. Sending a text that says fail is not good enough.